sensor_core/dsp/dsp_manager.py

```diff
@@ -3,59 +3,87 @@
 import scipy.signal as signal
 
 class DSPManager(DictManager):
-    def _init_(self, dsp_module_queue: np.array([]), dsp_modules: dict = None):
+    def _init_(self, dsp_modules_queue: np.array([]), dsp_modules: dict = None):
```

Comment:
Since you're anyway defining dsp_modules and dsp_modules_queue upon initialization, why are they passed as initialization parameters?